charts/minecraft/templates/deployment.yaml

```diff
@@ -204,7 +204,9 @@ spec:
         {{- if .projects }}
 {{- template "minecraft.envMap" list "MODRINTH_PROJECTS" (join "," .projects) }}
         {{- end }}
-        {{- if .optionalDependencies }}
+        {{- if .downloadDependencies }}
+{{- template "minecraft.envMap" list "MODRINTH_DOWNLOAD_DEPENDENCIES" .downloadDependencies }}
+        {{- else if .optionalDependencies }}
 {{- template "minecraft.envMap" list "MODRINTH_DOWNLOAD_OPTIONAL_DEPENDENCIES" .optionalDependencies }}
```

Comment:
While you're in here can you remove all these references to MODRINTH_DOWNLOAD_OPTIONAL_DEPENDENCIES? That variable doesn't exist anymore.